fe/fe-core/src/main/java/com/starrocks/lake/StarMgrMetaSyncer.java

```diff
@@ -284,7 +285,8 @@ public void syncTableMetaAndColocationInfo() {
     }
 
     // return true if starmgr shard meta changed
-    private boolean syncTableMetaInternal(Database db, OlapTable table, boolean forceDeleteData) throws DdlException {
+    @VisibleForTesting
+    public boolean syncTableMetaInternal(Database db, OlapTable table, boolean forceDeleteData) throws DdlException {
```

Comment:
remove the `public` accessor, the interface will be accessible inside the same package and will not be accessible outside the package.